src/test/java/test/reports/issue2415/IssueTest.java

```diff
@@ -0,0 +1,14 @@
+package test.reports.issue2415;
+
+import org.testng.TestNG;
+import org.testng.annotations.Test;
+import test.SimpleBaseTest;
+
+public class IssueTest extends SimpleBaseTest {
+    @Test
+    public void issueTest() {
```

Comment:
@bj-9527 - There are no assertions in this test. We should be checking the test results of all the methods and ensuring that after the first failure, all the other tests are all marked as `SKIPPED`